pycodestyle.py

```diff
@@ -1136,10 +1136,6 @@ def module_imports_on_top_of_file(
     Okay: # this is a comment\nimport os
     Okay: '''this is a module docstring'''\nimport os
     Okay: r'''this is a module docstring'''\nimport os
-    Okay:
-    try:\n\timport x\nexcept ImportError:\n\tpass\nelse:\n\tpass\nimport y
-    Okay:
-    try:\n\timport x\nexcept ImportError:\n\tpass\nfinally:\n\tpass\nimport y
```

Comment:
these were (surprise surprise!) not being tested -- when tested they produced other unrelated errors (tabs, etc.)

fortunately these are already tested in the E40 file